documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -170,18 +172,45 @@ stay in the cache or _maximum idle time_ , time it's allowed to be untouched
 
 .  _Expiration_ is a top-level construct, represented in the configuration as well as in the cache API.
 .  While eviction is _local to each cache instance_ , expiration is _cluster-wide_ .  Expiration lifespans and maxIdle values are replicated along with the cache entry.
-.  While maxIdle is replicated, expiration due to maxIdle is not cluster wide, only lifespan.  As such it is not recommended to use maxIdle in a clustered cache.
+.  Max idle requires additional network messages to work in a cluster.  As such using maxIdle in a clustered cache can cause some operations to be slower at times.
 .  Expiration lifespan and maxIdle are also persisted in CacheStores, so this information survives eviction/passivation.
 
+
+==== Max Idle Details
+
+Max idle expiration occurs when an entry has not been access withing a given time frame. This poses a more interesting problem than lifespan, which is very simple. Thus this type of expiration behaves slightly different in a local versus cluster cache.
+
+===== Local Max Idle
+
+Max idle is pretty straight forward in a non clustered cache. It removes expired entries when accessed directly (ie. Cache.get) and also removes expired entries entries when iterated upon (ie. Cache.size).
+
+===== Cluster Max Idle
+
+Max idle in a cluster is quite a bit more complicated. Since each node can have a different access time, an entry is not removed until all owners of said entry say it hasn't been accessed in the given amount of time.
+
+This check is done when a given node finds an entry has expired via maxIdle. It then will ask all the other owners what their last access time was (if it hasn't expired). If all owners say that the entry has expired, then it will not be returned to the user and removed based on the operation.
+
+NOTE: Clustered cache should always enable reaper when using max idle expiration due to some limitations listed in the section below.
+
+This points here highlight some useful information when using max idle.
```

Comment:
"When using max idle, it is important to keep the following points in mind:"

Reply:
👍